src/Nethermind/Nethermind.Mev/IMevConfig.cs

```diff
@@ -24,5 +26,25 @@ public interface IMevConfig : IConfig
             Description = "Defines whether the MEV bundles are allowed.",
             DefaultValue = "false")]
         bool Enabled { get; set; }
+
+        [ConfigItem(
+            Description = "Defines how long MEV bundles will be kept in memory by clients", 
+            DefaultValue = "3600")]
+        UInt256 BundleHorizon { get; set; }
+        
+        [ConfigItem(
+            Description = "Defines the maximum number of MEV bundles that can be kept in memory by clients", 
+            DefaultValue = "200")]
+        int BundlePoolSize { get; set; }
```

Comment:
Ideally things like this would be added at the end when the important functionality is delivered.